xml/System.Net/WebHeaderCollection.xml

```diff
@@ -82,8 +82,10 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Net.WebHeaderCollection> class is generally accessed through <xref:System.Net.WebRequest.Headers%2A?displayProperty=nameWithType> or <xref:System.Net.WebResponse.Headers%2A?displayProperty=nameWithType>. Some common headers are considered restricted and are either exposed directly by the API (such as `Content-Type`) or protected by the system and cannot be changed.  
-  
+ The <xref:System.Net.WebHeaderCollection> class is generally accessed through <xref:System.Net.WebRequest.Headers%2A?displayProperty=nameWithType> or <xref:System.Net.WebResponse.Headers%2A?displayProperty=nameWithType>.
+
+ On .NET Framework, some common headers are considered restricted and are either exposed directly by the API (such as `Content-Type`) or protected by the system and cannot be changed. This does *not* apply to .NET Core versions through version 3.1, .NET 5.0 and later versions.
```

Comment:
```suggestion
 On .NET Framework, some common headers are considered restricted and are either exposed directly by the API (such as `Content-Type`) or protected by the system and cannot be changed. This does *not* apply to .NET Core - none of the headers are restricted there.
```

Reply:
I'd leave out the .NET Core versions IMO -- it is missing from the time the APIs were ported to .NET Core -- whichever the version was.

Reply:
I followed the pattern from when I described max header length check behavior change here https://github.com/dotnet/dotnet-api-docs/pull/4642 but I see now that current situation is a bit different (for max length check, it also changed since 5.0, that's why .NET Core version was explicitly specified) -- but I'd still leave ".NET Core, .NET 5.0 and later versions" to avoid confusion